File: 1/1.2/Go/1.2.go
package main

import (
	"fmt"
	"strings"
)

func isAnagram(string1 string, string2 string) bool {
	if len(string1) != len(string2) {
		return false
	}

	for _, c := range string1 {
		if strings.Count(string1, string(c)) != strings.Count(string2, string(c)) {
			return false
		}
	}
	return true
}

func main() {
	fmt.Println(isAnagram("hello", "hell"))
	fmt.Println(isAnagram("hello", "ehllo"))
}

File: 1/1.2/Python/1.2.py
def isAnagram(string1: str, string2: str) -> bool:
    if len(string1) != len(string2):
        return False

    for c in string1:
        if string1.count(c) != string2.count(c):
            return False

    return True


if __name__ == '__main__':
    print(isAnagram("hello", "hell"))
    print(isAnagram("hello", "ehllo"))